Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM python:3.9.5
+FROM python:3.10.4
 
-ENV INDICO_HOST="dev.indico.io"
+ENV INDICO_HOST="dev-ci.us-east-2.indico-dev.indico.io"
```

Comment:
do we need to update this?

Reply:
good to do for tests I suppose

Reply:
yeah I figured it would be good if integration tests are run in deploy step